eng/Version.Details.xml

```diff
@@ -286,6 +286,15 @@
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>c07d59af4edfb22ebef9a8922cca990f6242b53b</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.NET.Runtime.MonoAOTCompiler.Task" Version="6.0.0-preview.2.21116.2">
+      <Uri>https://github.com/dotnet/runtime</Uri>
+      <Sha>c07d59af4edfb22ebef9a8922cca990f6242b53b</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.NET.Runtime.WebAssembly.Sdk" Version="6.0.0-preview.2.21116.2">
+      <Uri>https://github.com/dotnet/runtime</Uri>
+      <Sha>c07d59af4edfb22ebef9a8922cca990f6242b53b</Sha>
+    </Dependency>
```

Comment:
Why do you need that if you already have darc runtime dependency?

Reply:
I can check with the build team, but the file usually lists every dependency individually even if they're produced from the same build pipeline.